CODE_OF_CONDUCT.md

```diff
@@ -1,71 +1,57 @@
-Contributor Covenant Code of Conduct
+# Contributor Covenant Code of Conduct
 
-Our Pledge
+## Our Pledge
 
-In the interest of fostering an open and welcoming environment, we as
-contributors and maintainers pledge to making participation in our project and
-our community a harassment-free experience for everyone, regardless of age, body
-size, disability, ethnicity, gender identity and expression, level of experience,
-nationality, personal appearance, race, religion, or sexual identity and
-orientation.
+In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of age, body size, disability, ethnicity, gender identity and expression, level of experience, nationality, personal appearance, race, religion, or sexual identity and orientation.
 
-Our Standards
+---
```

Comment:
These horizontal rules seem gratuitous. Adding headings is enough. Suggest removing. Apologies for not catching this in initial review.